OlexanderHramatovych/HW5/Task2.py

```diff
@@ -0,0 +1,11 @@
+users_number = int(input("Type your number: "))
```

Comment:
good job
but I recommend using more meaningful variable names
look at this version of code
# Prompting the user to enter a number
users_number = int(input("Type your number: "))

# Initializing variables for the sequence
previous_term = 0
current_term = 1

# Generating and printing the sequence up to the user's number
while current_term <= users_number:
    print(current_term, end=", ")
    
    # Updating terms for the next iteration
    previous_term, current_term = current_term, previous_term + current_term

print("...")